### PR TITLE
import: log L0 SST ranges when too many files in ingest

### LINKED ISSUE #13980: log L0 SST ranges when ImportSstService found too many files
### Is your feature request related to a problem? Please describe:

to help understand what causes so many files by checking the start/end key

### PR DESCRIPTION
**Pull request by lance6716**
Signed-off-by: lance6716 <lance6716@gmail.com>

### What is changed and how it works?

Issue Number: Close #13980

What's Changed:

```commit-message
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

Side effects

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by gozssky — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]
LGTM

**Comment by sticnarf** [comment]
I don't know about the background. Do you already have some suspecion about what abnormal L0 SST ranges would look like and what can we do after we know the ranges?

**Comment by lance6716** [comment]
> I don't know about the background. Do you already have some suspecion about what abnormal L0 SST ranges would look like and what can we do after we know the ranges?

We suspect memtable covers a large range so a very "flat" SST file is flushed, blocking the futher ingestion. This PR want to print some clue about it, maybe we can see metaKV and rowKV is the start/end key of a SST file so indexKV ingestion is blocked.

File: src/import/sst_service.rs [hunk 1]
```diff
@@ -185,6 +185,17 @@ where
                 .ingest_maybe_slowdown_writes(CF_WRITE)
                 .expect("cf")
         {
+            match self.engine.get_sst_key_ranges(CF_WRITE, 0) {
+                Ok(l0_sst_ranges) => {
+                    warn!(
+                        "sst ingest is too slow";
+                        "sst_ranges" => ?l0_sst_ranges,
```
Comment (sticnarf) [inline src/import/sst_service.rs:192]: Do you want to use log_wrapper to convert it to hex and add auto-redaction support?

Comment (lance6716) [reply src/import/sst_service.rs:192]: checked that in order to know the problem we can't use redact 😂

**Review by sticnarf — APPROVED** [review]

**Comment by lance6716** [comment]
/assign @sticnarf 

no privilege, please help merge it

**Pull request merged**